testsuite/features/step_definitions/command_steps.rb

```diff
@@ -580,12 +580,17 @@
     raise ScriptError, 'File injection failed' unless success
   end
   server.run('chmod +x /etc/ipmi/fake_ipmi_host.sh', verbose: true, check_errors: true)
-  server.run('ipmi_sim -n < /dev/null > /dev/null &', verbose: true, check_errors: true)
+  # Check if ipmi_sim is already running
+  if server.run('pgrep -f ipmi_sim', verbose: false, check_errors: false).empty?
+    server.run('ipmi_sim -n < /dev/null > /dev/null &', verbose: true, check_errors: true)
+  else
+    puts 'ipmi_sim is already running; skipping startup.'
```

Comment:
I would better use `log` method instead of `puts`, which is handled by Capybara.